libpldmresponder/fru.cpp

```diff
@@ -321,7 +340,20 @@ void FruImpl::populateRecords(
             if (numRecs == numRecsCount)
             {
                 recordSetIdentifier = nextRSI();
-                bmc_record_handle = nextRecordHandle();
+                if (concurrentAdd)
+                {
+#ifdef OEM_IBM
+                    auto lastLocalRecord = pldm_pdr_find_last_in_range(
+                        pdrRepo, BMC_PDR_START_RANGE, BMC_PDR_END_RANGE);
+                    bmc_record_handle =
+                        pldm_pdr_get_record_handle(pdrRepo, lastLocalRecord) +
+                        1;
+#endif
```

Comment:
need to use oem-ibm handler with the code moved to oem, no if-defs.

Reply:
done